onnxruntime/core/graph/contrib_ops/quantization_defs.cc

```diff
@@ -131,149 +116,208 @@ Performs element-wise binary {name} on 8 bit data types (with Numpy-style broadc
+                                      "If it's specified, it means per 'axis' quantization and input 'x_scale' and "
+                                      "'x_zero_point' must be 1-D tensors.",
+                                      AttributeProto::INT, false)
+                                .Input(0, "x", "N-D quantized Input tensor to be de-quantized.", "T1")
+                                .Input(1, "x_scale",
+                                       "Scale for input 'x'. It could be a scalar or a 1-D tensor, which means a "
+                                       "per-tensor or per-axis quantization."
+                                       "If it's a 1-D tensor, its number of elements should be equal to the dimension "
+                                       "value of 'axis' dimension of input 'x'.",
+                                       "T2")
+                                .Input(2, "x_zero_point",
+                                       "Zero point for input 'x'. It could be a scalar or a 1-D tensor, which means a "
+                                       "per-tensor or per-axis quantization."
+                                       "If it's a 1-D tensor, its number of elements should be equal to the dimension "
+                                       "value of 'axis' dimension of input 'x'.",
+                                       "T1")
+                                .Output(0, "y", "N-D full precision output tensor. It has same shape as input 'x'.",
+                                        "T2")
+                                .TypeConstraint("T1", {"tensor(int8)", "tensor(uint8)"},
+                                                "Constrain 'x' and 'x_zero_point' to 8-bit integer tensors.")
+                                .TypeConstraint("T2", {"tensor(float16)", "tensor(float)"},
+                                                "Constrain 'y', 'x_scale' to float tensors.")
+                                .SetDoc(DequantizeLinear_ver1_doc)
+                                .TypeAndShapeInferenceFunction([](ONNX_NAMESPACE::InferenceContext& ctx) {
+                                  auto y_type = ctx.getOutputType(0);
+                                  // only float is supported
+                                  y_type->mutable_tensor_type()->set_elem_type(ONNX_NAMESPACE::TensorProto::FLOAT);
+
+                                  if (!hasInputShape(ctx, 0)) return;
+
+                                  auto& input_shape = getInputShape(ctx, 0);
+                                  updateOutputShape(ctx, 0, input_shape);
+                                }));
+
+static const char* QuantizeBFP_ver1_doc = R"DOC(
+The BFP quantization operator. It consumes a full precision tensor and computes an BFP tensor.)DOC";
+
+ONNX_MS_OPERATOR_SET_SCHEMA(
+    QuantizeBFP, 1,
+    OpSchema()
+        .Attr("bfp_type", "The type of BFP - must match with the BFPType enum", AttributeProto::INT)
+        .Attr("block_dims",
```

Comment:
@drwebb I've renamed and updated the description. Please re-open if you have any concerns.